### PR TITLE
Extend Stringutils with padRight

### PR DESCRIPTION
**Pull request by lsiepel**
Earlier some string utils where added to core in order to remove the dependency on `org.apache.commons.lang3.StringUtils`.
While doing so i create a `padLeft`, but forgot a `padRight` method. This PR corrects that.

### CODE REVIEW

File: bundles/org.openhab.core/src/main/java/org/openhab/core/util/StringUtils.java [hunk 1]
```diff
@@ -192,6 +192,25 @@ public static String padLeft(@Nullable String str, int minSize, String padString
...
+      * padRight("openHAB", 4, "*")  => "openHAB"
+     * </pre>
+     *
+     * @param str the String to pad, may be null
+     * @param minSize the minimum String size to return
+     * @param padString the String to add when padding
+     * @return the padded String
+     */
+    public static String padRight(@Nullable String str, int minSize, String padString) {
+        String paddedString = str == null ? "" : str;
```
Comment (J-N-K) [inline bundles/org.openhab.core/src/main/java/org/openhab/core/util/StringUtils.java:210]:
```suggestion
        String paddedString = Objects.requireNonNullElse(str, "");
```

**Review by J-N-K — CHANGES_REQUESTED** [review]

**Review by holgerfriedrich — COMMENTED** [review]
I hope I do not get this wrong, but I there are two observations (for edge cases):
- It seems you replace all occurrences of ```" "``` in the original String as well.
- You replace a single character ```" "``` possibly by a multi-character String, which might result in the String to be longer than minSize.

Both might be unexpected for the user.

File: bundles/org.openhab.core/src/main/java/org/openhab/core/util/StringUtils.java (hunk 1 above)
Comment (lsiepel) [reply bundles/org.openhab.core/src/main/java/org/openhab/core/util/StringUtils.java]: Fixed

**Comment by lsiepel** [comment]
> I hope I do not get this wrong, but I think there are two observations (for edge cases):
> 
> * It seems you replace all occurrences of `" "` in the original String as well.
> * You replace a single character `" "` possibly by a multi-character String, which might result in the String to be longer than minSize.
> 
> Both might be unexpected for the user.

Agree that might be unexpected, i just changed the implementation a bit, support for strings with whitespaces is added now.

**Review by holgerfriedrich — APPROVED** [review]
LGTM, thanks

**Review by J-N-K — APPROVED** [review]
Thanks!

**Pull request merged**